lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -188,7 +199,8 @@ def test_scatter3d_color():
                color='b', marker='s')
 
 
-@image_comparison(['plot_3d_from_2d.png'], remove_text=True)
+@image_comparison(['plot_3d_from_2d.png'],
+                  remove_text=True, style='default', tol=0.01)
```

Comment:
@anntzer, fix was to add this `tol` argument, as mentioned in the commit. Actual RMS was `0.008`, so this seemed a reasonable choice